### PR TITLE
projects: Add 'optimize caching' project

### PR DESCRIPTION
**Pull request by fneu**

### CODE REVIEW

File: data/projects.js [hunk 1]
```diff
@@ -81,5 +81,27 @@ coala should support generating metrics for your code.",
...
 		"tags" : ["Core", "Bears"],
 		"markdown": "implement_metrics_for_coala.md"
+	},
+	{
+		"name" : "Optimize Caching",
+		"desc" : "Improve coala's performance by implementing caching strategies.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs.",
+			"The student must have proved his or her throughout understanding of the coala core."
```
Comment (sils) [inline data/projects.js]: prove it through the aplication? We gotta be careful, google doesn't like orgs who sponsor just their experienced members. GSoC is a newcomer program.

Comment (fneu) [inline data/projects.js]: yeah that's probably a little harsh.

Comment (fneu) [inline data/projects.js]: "The proposal needs to show some familiarity with coala's caching implementation"

or your typical

"The proposal needs to contain a clear concept." ?

**Comment by sils** [comment]
@fneu you got conflicts

**Comment by sims1253** [comment]
https://github.com/coala/projects/pull/89 took this

**Pull request closed without merging**